app/src/main/java/com/dsciitp/shabd/Dictionary/DictionaryActivity.java

```diff
@@ -1,39 +1,27 @@
 package com.dsciitp.shabd.Dictionary;
 
-import android.net.Uri;
-import android.os.AsyncTask;
 import android.os.Bundle;
 import android.speech.tts.TextToSpeech;
 import android.support.design.widget.FloatingActionButton;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
 import android.support.v7.app.ActionBar;
 import android.support.v7.app.AppCompatActivity;
-import android.support.v7.widget.GridLayoutManager;
-import android.support.v7.widget.RecyclerView;
-import android.util.Log;
 import android.view.View;
 import android.widget.EditText;
 import android.widget.ImageView;
-import android.widget.TextView;
 import com.dsciitp.shabd.R;
-import org.json.JSONException;
-import org.json.JSONObject;
-import java.io.IOException;
-import java.net.MalformedURLException;
-import java.net.URL;
-import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Locale;
 
-public class DictionaryActivity extends AppCompatActivity implements DictionaryAdapter.OnCategorySelectedListener {
+public class DictionaryActivity extends AppCompatActivity implements DictionaryAdapter.OnCategorySelectedListener, DictionaryFragment.OnCategorySelectedListener {
 
     EditText word;
-    TextView meaning;
     ImageView del;
     TextToSpeech t1;
     View v;
+    FloatingActionButton fab1, search;
     private static final String TTS_SPEAK_ID = "SPEAK";
-    String base = "https://googledictionaryapi.eu-gb.mybluemix.net";
-    final static String PARAM_QUERY = "define";
+    String press = "Press me to know the meaning";
```

Comment:
What is the use of this hard coded string?

Reply:
It's logic will be implemented later  with use of a string resource for this

Reply:
okkk